docs/app/Examples/modules/Checkbox/States/CheckboxRemoteControlExample.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {Button, Checkbox} from 'stardust';
+
+export default class CheckboxRemoteControlExample extends Component {
+  toggle = () => {
+    this.refs.checkbox.plugin('toggle');
+  };
```

Comment:
Are we going to include these methods inside the stardust docs? Or just refer them to the semantic docs?